### ISSUE TITLE
Destination Signs

### ISSUE THREAD

**Issue by mfbehrens99** [osmandapp/Osmand#9142]
Displaying destinations_sign Relations would be a pretty nice feature.

Everything is documented here: https://wiki.openstreetmap.org/wiki/Relation:destination_sign

A good example for a guidepost with many Relations for testing is Relation [7590738631](https://osm.org/node/7590738631)

**Comment by vmicho** [comment]
Yes, especially for guideposts would be usefull. Cuz unless they have destination_xxx tags, they seem to be empty and I usually have to re-photo them :D

Example:
https://hiking.waymarkedtrails.org/#guidepost?id=2654545863&map=18.0/46.966/7.4386
(you can click on destinations, then more details to see it on other website) -> these are taken from the "destination_sign" relation, the direction pointer is approximated from the "intersection" point and way "to"
